common/ethtool.go

```diff
@@ -0,0 +1,51 @@
+package common
+
+import "fmt"
+import "syscall"
+import "unsafe"
+
+const (
+	SIOCETHTOOL     = 0x8946     // linux/sockios.h
+	ETHTOOL_STXCSUM = 0x00000017 // linux/ethtool.h
+)
+
+// linux/if.h 'struct ifreq'
+type IFReqData struct {
+	Name [16]byte
+	Data uintptr
+}
+
+// linux/ethtool.h 'struct ethtool_value'
+type EthtoolValue struct {
+	Cmd  uint32
+	Data uint32
+}
+
+// Disable TX checksum offload on specified interface
+func EthtoolTXOff(name string) error {
+	if len(name) > 15 {
```

Comment:
Yes, but possibly more related to the [16] earlier.
Which comes from http://lxr.free-electrons.com/source/include/uapi/linux/if.h#L28
I will consider how to combine them.